apps/products-feed/src/modules/metadata-cache/update-cache-for-configurations.ts

```diff
@@ -0,0 +1,35 @@
+import { createLogger } from "@saleor/apps-shared";
+import { CacheConfigurator } from "./cache-configurator";
+import { createSettingsManager } from "../../lib/metadata-manager";
+import { getCursors } from "../../lib/google-feed/fetch-product-data";
+import { Client } from "urql";
+import { z } from "zod";
+import { appConfigInputSchema } from "../app-configuration/app-config-input-schema";
+
+interface UpdateCacheForConfigurationsArgs {
+  client: Client;
+  saleorApiUrl: string;
+  configurations: z.infer<typeof appConfigInputSchema>;
+}
+
+export const updateCacheForConfigurations = async ({
+  client,
+  configurations,
```

Comment:
now its fine but it can be optimized - when webhook is triggered for a product, we can check what was the channel of this product and invalidate only it